apis/config/types_landscaper_config.go

```diff
@@ -45,8 +45,23 @@ type LandscaperConfiguration struct {
 
 // LsDeployments contains the names of the landscaper deployments.
 type LsDeployments struct {
+	// LsController is the name of the Landscaper controller deployment.
 	LsController string
```

Comment:
only a remark: in the future we will have an additional deployment for the inst and exec controllers